apps/roam/src/components/settings/Settings.tsx

```diff
@@ -94,26 +89,38 @@ export const SettingsDialog = ({
           selectedTabId={selectedTabId}
           vertical={true}
           renderActiveTabPanelOnly={true}
-          className="h-full"
         >
+          <div className="mb-2 mt-6 text-lg font-semibold text-neutral-dark">
```

Comment:
Let's nix the `mt-6`.  Let's try to align it with the tab.
![image](https://github.com/user-attachments/assets/a5b9b509-6545-414f-b17e-83c56d4cc44e)